### PR TITLE
test: moving test from aqua to fluence cli

### PR DESCRIPTION
**Pull request by gurinderu**
## Description 

- Changed aqua cli to fluence cli
- Changed key generation for the new cli

### CODE REVIEW

**Review by nahsi — APPROVED** [review]

File: integration-tests/framework.py [hunk 1]
```diff
@@ -1,15 +1,34 @@
+import string
+
+import filelock
 import pytest
 import time
 import delegator
 import random
 import json
-import ed25519
+import string
```
Comment (justprosh) [inline integration-tests/framework.py:9]: `string` imported twice

File: integration-tests/framework.py [hunk 2]
```diff
@@ -63,15 +86,18 @@ def from_aqua(aqua, func_name):
             raise Exception(f"Unable to read compiled air script by path {air_path}: {e}")
         return air_script
 
-# TODO: learn how to chose the relay based on the test worker id.
-def run_aqua(sk, func, args, relay=get_relay()):
 
+# TODO: learn how to chose the relay based on the test worker id.
```
Comment (justprosh) [inline integration-tests/framework.py:90]:
```suggestion
# TODO: learn how to choose the relay based on the test worker id.
```

File: integration-tests/framework.py [hunk 3]
```diff
@@ -182,10 +226,13 @@ def setup_class(cls):
...
     # update teardown_class to remove a sepll + calling the original one
     old_teardown_class = getattr(cls, "teardown_class", None)
+
     def teardown_class(cls):
         if old_teardown_class is not None:
             old_teardown_class()
-        destroy_spell(cls.sk, cls.spell_id)
+        destroy_spell(cls.key_pair_name, cls.spell_id)
+        delete_key(cls.key_pair_name)
+        delete_key(cls.key_pair_name)
```
Comment (justprosh) [inline integration-tests/framework.py:235]: why you delete key twice?

**Review by justprosh — APPROVED** [review]

File: integration-tests/test_aqua.py [hunk 4]
```diff
@@ -1,29 +1,34 @@
...
     return {
-      "clock":       { "start_sec": 0, "end_sec": 0, "period_sec": 0 },
-      "connections": { "connect": False, "disconnect": False},
-      "blockchain":  { "start_block": 0, "end_block": 0 }
+        "clock": {"start_sec": 0, "end_sec": 0, "period_sec": 0},
+        "connections": {"connect": False, "disconnect": False},
+        "blockchain": {"start_block": 0, "end_block": 0}
     }
 
+
```
Comment (kmd-fl) [inline integration-tests/test_aqua.py:12]: Is it automated formatting?

File: integration-tests/test_aqua.py [hunk 5]
```diff
@@ -87,25 +93,28 @@ def test_install_spell_id(self):
         assert len(self.spell_id) != 0
 
     def test_install_get_config(self):
-        cfg_result = run_aqua(self.sk, "get_config", [self.spell_id])
+        cfg_result = run_aqua(self.key_pair_name, "get_config", [self.spell_id])
         assert cfg_result["success"]
-        assert cfg_result["config"] == self.config, "spell's config should be equal the one that was set during installtion"
+        assert cfg_result[
```
Comment (kmd-fl) [inline integration-tests/test_aqua.py:98]: Is it also autoformatting? Not a very intuitive place to add a newline.

File: integration-tests/framework.py (hunk 1 above)
Comment (gurinderu) [reply integration-tests/framework.py:7]: Fixed

File: integration-tests/framework.py (hunk 2 above)
Comment (gurinderu) [reply integration-tests/framework.py]: Fixed

File: integration-tests/framework.py (hunk 3 above)
Comment (gurinderu) [reply integration-tests/framework.py]: Fixed

File: integration-tests/test_aqua.py (hunk 4 above)
Comment (gurinderu) [reply integration-tests/test_aqua.py]: Fixed

File: integration-tests/test_aqua.py (hunk 5 above)
Comment (gurinderu) [reply integration-tests/test_aqua.py]: Fixed

File: integration-tests/test_aqua.py [hunk 6]
```diff
@@ -244,41 +256,39 @@ class TestTriggerMailbox:
     dat = {}
 
     def test_triggers_oneshot(self):
-        trigger = get_trigger_event_ok(self.sk, self.spell_id)
+        trigger = get_trigger_event_ok(self.key_pair_name, self.spell_id)
 
-        assert len(trigger) == 0, "no triggers must be in the spell's trigger mailbox on empty config"
+        assert trigger is None, "no triggers must be in the spell's trigger mailbox on empty config"
```
[the remark this answers is not in the document]
Comment (gurinderu) [reply integration-tests/test_aqua.py:249]: the same story(

File: integration-tests/framework.py [hunk 7]
```diff
@@ -83,61 +109,78 @@ def run_aqua(sk, func, args, relay=get_relay()):
...
+    result = update_spell(key_pair_name, spell_id, config)
     assert result["success"], f"can't update the spell {spell_id}"
 
-def get_trigger_event_ok(sk, spell_id):
-    [trigger, error] = run_aqua(sk, "get_trigger_event", [spell_id])
-    assert len(error) == 0, f"get_trigger_event: got error while retrieving triggers for spell {spell_id}: {error}"
+
+def get_trigger_event_ok(key_pair_name, spell_id):
+    [trigger, error] = run_aqua(key_pair_name, "get_trigger_event", [spell_id])
+    assert error is None, f"get_trigger_event: got error while retrieving triggers for spell {spell_id}: {error}"
```
[the remark this answers is not in the document]
Comment (folex) [reply integration-tests/framework.py:160]: `null` seems fine according to Aqua signatures.

**Review by folex — APPROVED** [review]

**Pull request merged**